src/raster/r.damflood/r.damflood.html

```diff
@@ -1,140 +1,197 @@
+The numerical model solves the conservative form of the shallow water equations
+(SWE) using a finite volume method (FVM);
+the intercell flux is computed by the "upwind method and the water-level
+gradient is evaluated by weighted average of both upwind and downwind gradient. 
+Additional details of the specific numerical scheme adopted in the model are
 presented in references [1].<br>
 
-The command allows to generate raster time series, of water depth and flow velocity, with time resolution defined by user. 
-Each time series is identified by a number of raster maps named with a common prefix as specified by the user and the time instant
-which it refers expressed in seconds from the dam failure, joined by the underscore character (e.g.; myvel_125, myvel_250, myvel_375, etc.).<br>
-
-Because this new module has been implemented with the aim to provide an instrument for risk assessment fully within a GIS environment,
-it should be able to provide intensity maps directly applicable in those analyses.In floods, intensity generally corresponds 
-to the maximum flow depth, but in the particular case of flash floods, where velocities are normally high,
-it is recommended to use as intensity indicator the maximum between the water depth and the product of water velocity and water depth. 
-For this reason, with this module, in addition to the water depth and velocity maps, the user can choose
-a variety of output raster maps: maximum water depth, maximum water velocity, and maximum intensity raster maps. <br>
-
-In case on high numerical stability problem, the user is warned, and the simulation is stopped.<br>
-<br><br>
+The module allows to generate raster time series, of water depth and
+flow velocity, with time resolution defined by the user.
+Each time series is identified by a number of raster maps named with a common
+prefix as specified by the user and the time instant which it refers 
+expressed in seconds from the dam failure, joined by the underscore character 
+(e.g.; myvel_125, myvel_250, myvel_375, etc.).<br>
+
+Because this new module has been implemented with the aim to provide an 
+instrument for risk assessment fully within a GIS environment, it should be 
+able to provide intensity maps directly applicable in those analyses.
+In floods, intensity generally corresponds to the maximum flow depth, 
+but in the particular case of flash floods, where velocities are normally high,
+it is recommended to use as an intensity indicator the maximum between
+the water depth and the product of water velocity and water depth.
+For this reason, with this module, in addition to the water depth and velocity
+maps, the user can choose a variety of output raster maps: maximum water depth, 
+maximum water velocity, and maximum intensity raster maps.<br>
+
+In case of numerical stability problems, the user is warned, and the
+simulation is stopped.<br>
```

Comment:
I had a hard time deciding if this was the correct interpretation, given that code is speaking of "Courant-Friedrich-Lewy stability condition" and seems specific to the numerical algorithm. I tried to learn more on this, but I'm still not quite sure. This sentence refers to (now) lines 889-895 of main.c,  comments at (now) lines 84-94 of SWE.c, code and comments of lines 550-536 and 724-742 of SWE.c.